tpp-app/src/services/SettingsService.js

```diff
@@ -96,6 +118,24 @@ export const POSTRemindLogPeriod = async (enableRemind) => new Promise(async (re
     }
 });
 
+/**
+ * Retrieves whether the user wants a remind to log period
+ * @returns a promise resolving in a boolean when the get operation is complete
+ */
+export const GETRemindLogPeriod = async () => new Promise(async(resolve, reject) => {
```

Comment:
here try/catch makes sense since there's no .catch() but also it would be better if you matched the structure to POSTUpdateOnePreference. either have both functions use try/catch and no .catch() or have both use no try/catch and .catch() 

Also doesn't need await since you have .then(), and doesn't need ```value = ``` since value is never referenced. What you are resolving is the value from .then(value) so you don't need the value variable from ```value = ```

This should be good: 
```
await AsyncStorage.getItem(REMINDERS.REMIND_LOG_PERIOD
            ).then((value) => {
                console.log(`Retrieved RemindLogPeriod boolean`);
                resolve(value != null ? JSON.parse(value) : null);
            })
```